README.md

```diff
@@ -8,11 +8,17 @@
 [![codecov](https://codecov.io/gh/procyon-rs/vega_lite_3.rs/branch/master/graph/badge.svg)](https://codecov.io/gh/procyon-rs/vega_lite_3.rs)
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=procyon-rs/vega_lite_3.rs)](https://dependabot.com)
 
-A Rust api for vega-lite v3. Use it to generate vega-lite json, to display result in your Browser or in Jupyter (via [showata](https://crates.io/crates/showata)).
+## About
```

Comment:
As you're changing the about, could you add a link to Vega itself, and maybe a quick description of what it is?
We're actually never explaining that, and it's bad for a newcomer

Reply:
Yes I will do so and also to vega-lite

Reply:
Maybe you can remove the title `## About`